OmiseSwift/API Models/Source/SourceType.swift

```diff
@@ -55,6 +57,8 @@ public enum InternetBanking: RawRepresentable, Equatable, Hashable {
 public enum SourceType: Codable, Equatable, Hashable {
     case internetBanking(InternetBanking)
     case alipay
+    case promptpay
+    case paynow
```

Comment:
Shall we name these in the camel case? @igroomgrim

Reply:
Unfortunately right now it's a little bit mixed. TrueMoney is also not named in camel case. But most other source types are named in camel case so it probably makes sense to use it here as well.

Reply:
Okay I changed the naming to camel case.

Reply:
it's ok for this case 👍